src/Vipps.net/Services/LoginService.cs

```diff
@@ -1,21 +1,50 @@
 ﻿using System;
 using System.Net.Http;
-using System.Security.Cryptography;
 using System.Threading;
 using System.Threading.Tasks;
-using Polly.Retry;
 using Vipps.net.Infrastructure;
 using Vipps.net.Models.Login;
 
 namespace Vipps.net.Services
 {
+    public interface IVippsLoginService
```

Comment:
Imo should move these interfaces to their own file and remove `Vipps` from name so they're in line with the class names.

Reply:
Agree that they should have the same name, added "Vipps" to the classnames.